src/main/java/world/bentobox/bentobox/BentoBox.java

```diff
@@ -203,8 +203,8 @@ public void onEnable(){
 
             // Save islands & players data every X minutes
             Bukkit.getScheduler().runTaskTimer(instance, () -> {
-                playersManager.asyncSaveAll();
-                islandsManager.asyncSaveAll();
+                playersManager.saveAll();
+                islandsManager.saveAll();
             }, getSettings().getDatabaseBackupPeriod() * 20 * 60L, getSettings().getDatabaseBackupPeriod() * 20 * 60L);
```

Comment:
These methods were attempting to save the tables one entry per tick, but as database saving is now async, they should not be required. Further, the larger the number of entries, the longer this will take and could start overlapping with previous saves.